### PR TITLE
Feature/main page mine ntu card

### PR DESCRIPTION
**Pull request by jc-hiroto**
- Add cards to index page (main-page) for user to access different services.

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -38,6 +39,7 @@ export default {
 
   // Modules for dev and build (recommended): https://go.nuxtjs.dev/config-modules
   buildModules: [
+    '@nuxtjs/fontawesome'
```
Comment (maxchou415) [inline nuxt.config.js:42]: 沒有用的套件要刪掉 😂

File: nuxt.config.js [hunk 2]
```diff
@@ -18,7 +18,8 @@ export default {
     ],
     link: [
       { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' },
-      { rel: 'stylesheet', href: 'https://unpkg.com/tachyons@4.12.0/css/tachyons.min.css' }
+      { rel: 'stylesheet', href: 'https://unpkg.com/tachyons@4.12.0/css/tachyons.min.css' },
```
Comment (maxchou415) [inline nuxt.config.js:21]: 這也是

File: package.json [hunk 3]
```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@nuxtjs/auth-next": "5.0.0-1622309801.87126e3",
     "@nuxtjs/axios": "^5.13.1",
+    "@nuxtjs/fontawesome": "^1.1.2",
```
Comment (maxchou415) [inline package.json:14]: 刪刪刪

**Review by maxchou415 — CHANGES_REQUESTED** [review]

**Comment by jc-hiroto** [comment]
@maxchou415 deleted.

**Pull request closed without merging**